<launch>
<!-- Launch file for D-Map -->

	<arg name="rviz" default="true" />


	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find dmap)/rviz_config/dmap_velodyne.rviz" if="$(arg rviz)"/>

	<node pkg="dmap" type="example" name="DMap" output="screen">

        <rosparam command="load" file="$(find dmap)/config/velodyne.yaml" />
        <param name="map_res_min" type="double" value="0.25"/>                           <!-- Unit: meter -->
        <param name="map_res_init" type="double" value="5.0"/>                          <!-- Unit: meter -->
        <param name="full_ratio" type="double" value="0.9"/>                            <!-- Unit:   -   -->
        <param name="depthmap_accuracy" type="double" value="1.0"/>
        <param name="sliding_en" type="bool" value="false"/>
        <param name="sliding_thres" type="double" value="20.0"/>
        <param name="print_en" type="bool" value="false"/>
        <param name="log_map" type="bool" value="true"/>
        <param name="log_name" type="string" value="Kitti_test"/>
        <param name="vis_en" type="bool" value="$(arg rviz)"/>

	</node>


</launch>